tools/virtualization/offloading-domain-tty-output-to-console-with-custom-kernel-initramfs-images.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Offloading domain TTY output to console with custom kernel/initramfs images  
+subsection: virtualization  
+order: 12  
+---
+
+# Offloading domain TTY output to console with custom kernel/initramfs images
+
+1. Navigate to the `/boot` directory of the Fedora installation of the host to grab some kernel and initramfs images.  
+   ```
+   cd /boot && ls -l
+   ```
+   The following is an example output for the above command.  
+   ```
+   total 239596
+   -rw-r--r--. 1 root root   234567 Jul 21 02:13 config-5.13.4-200.fc34.x86_64
+   -rw-r--r--. 1 root root   234567 Jul 25 22:05 config-5.13.5-200.fc34.x86_64
+   -rw-r--r--. 1 root root   234567 Jul 28 21:17 config-5.13.6-200.fc34.x86_64
+   drwx------. 4 root root     4096 Jan  1  1970 efi
+   -rw-r--r--. 1 root root   155544 Jan 27  2021 elf-memtest86+-5.31
+   drwxr-xr-x. 2 root root     4096 Apr 23 16:26 extlinux
+   drwx------. 2 root root     4096 Aug  2 13:07 grub2
+   -rw-------. 1 root root 84860404 Jul 12 00:09 initramfs-0-rescue-8ee701c92f0f486cbb7afe96e5b7ea67.img
+   -rw-------. 1 root root 33023557 Jul 23 18:49 initramfs-5.13.4-200.fc34.x86_64.img
+   -rw-------. 1 root root 33002599 Jul 30 08:37 initramfs-5.13.5-200.fc34.x86_64.img
+   -rw-------. 1 root root 32910001 Aug  2 14:44 initramfs-5.13.6-200.fc34.x86_64.img
+   drwxr-xr-x. 3 root root     4096 Jul 12 00:09 loader
+   drwx------. 2 root root    16384 Jul 12 00:06 lost+found
+   -rw-r--r--. 1 root root   153868 Jan 27  2021 memtest86+-5.31
+   -rw-------. 1 root root  5778689 Jul 21 02:13 System.map-5.13.4-200.fc34.x86_64
+   -rw-------. 1 root root  5778727 Jul 25 22:05 System.map-5.13.5-200.fc34.x86_64
+   -rw-------. 1 root root  5778835 Jul 28 21:17 System.map-5.13.6-200.fc34.x86_64
+   -rwxr-xr-x. 1 root root 10580240 Jul 12 00:09 vmlinuz-0-rescue-8ee701c92f0f486cbb7afe96e5b7ea67
+   -rwxr-xr-x. 1 root root 10853744 Jul 21 02:13 vmlinuz-5.13.4-200.fc34.x86_64
+   -rwxr-xr-x. 1 root root 10851312 Jul 25 22:05 vmlinuz-5.13.5-200.fc34.x86_64
+   -rwxr-xr-x. 1 root root 10850800 Jul 28 21:17 vmlinuz-5.13.6-200.fc34.x86_64
+   ```
```

Comment:
I think this can be enhanced like so:
```suggestion
   ```console
   $ cd /boot
   $ ls -l vmlinuz-*
   -rwxr-xr-x. 1 root root 10580240 Jul 12 00:09 vmlinuz-0-rescue-8ee701c92f0f486cbb7afe96e5b7ea67
   -rwxr-xr-x. 1 root root 10853744 Jul 21 02:13 vmlinuz-5.13.4-200.fc34.x86_64
   -rwxr-xr-x. 1 root root 10851312 Jul 25 22:05 vmlinuz-5.13.5-200.fc34.x86_64
   -rwxr-xr-x. 1 root root 10850800 Jul 28 21:17 vmlinuz-5.13.6-200.fc34.x86_64
   [ . . . ]
   ```
```

Please check our Contributing guide: https://developer-portal.github.io/contributing/write-general-writing-hints.html

Reply:
Addressed here https://github.com/developer-portal/content/pull/394/commits/a72c21af900795cf46adb70c384502f395e19931